cypress/integration/pages/onDemandAudio/tests.js

```diff
@@ -60,9 +60,15 @@ export default ({ service, pageType, variant, isAmp }) => {
       });
       describe('Recent Episodes component', () => {
         it('should be displayed if the toggle is on, and shows the expected number of items', function test() {
+          let toggleName;
+          if (getDataUrl(Cypress.env('currentPath')).includes('podcasts')) {
```

Comment:
Don't think we need to get the data URL here.  Unless I'm missing something, `Cypress.env('currentPath')` will also include `podcasts` for podcast pages

```suggestion
          if (Cypress.env('currentPath')).includes('podcasts') {
```

Reply:
I think you are right. Will commit this change